lib/item_marketplace/presenters/item_presenter.rb

```diff
@@ -0,0 +1,66 @@
+class ItemPresenter
+  attr_reader :item
+
+  def initialize(item, bid = nil)
+    @item = item
+    @bid = bid
+  end
+
+  def errors
+    if @bid
+      @bid.errors.full_messages
+    end
+  end
+
+  def id
+    @item.id
+  end
+
+  def name
+    @item.name.capitalize
+  end
+
+  def description
+    @item.description
+  end
+
+  def buy_it_now_price
+    @item.buy_it_now_price
+  end
+
+  def starting_bid_price
+    @item.starting_bid_price
+  end
+
+  def highest_bid
+    @item.highest_bid
+  end
+
+  def duration
+    duration = (@item.auction_end_time - Time.now.utc).round
+  end
+
+  def days
+    duration/86400
+  end
+
+  def hours
+    (duration%86400)/3600
```

Comment:
There might be some nice Rails helpers for [this](http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words), but I also like what you have here. 👍